packages/medusa/src/models/product-category.ts

```diff
@@ -25,6 +25,9 @@ export class ProductCategory extends BaseEntity {
   @Column()
   name: string
 
+  @Column({ nullable: true, default: '' })
```

Comment:
updated

Reply:
Perfect, could we do a pr where you explicitly set the nullable values in order for sqlite to work as expected with the categories ?

Reply:
I don't follow. What do you mean?

Reply:
What I mean is that for example `parent_category_id` is nullable, but seams it is not specified on the column options, when you use sqlite the default value is nullable false, which breaks when you are using sqlite and not using the migration since it is based on entities sync

Reply:
so would be nice to specify all the nullable explicitly on the entity (e.g @Column({ nullable: true }))
Also, it goes along with the entity alignment with the migrations for those that are generating the migration automatically

Reply:
Hence Why I suggest a new pr for that in order to not extend this one

Reply:
Ah, I understand now. Thanks! Will create a ticket